app/assets/stylesheets/darkswarm/_shop-popovers.css.scss

```diff
@@ -25,54 +24,27 @@
 
   .joyride-nub.right {
     top: 38px;
-    border-color: #999 !important;
+    border-color: $grey-800 !important;
     border-top-color: transparent !important;
     border-right-color: transparent !important;
     border-bottom-color: transparent !important;
   }
 
-  .progress {
-    background-color: #13bf85;
-    padding: 0;
-    border: none;
-    color: white;
-    font-size: 0.75rem;
-    font-style: oblique;
-    line-height: 1;
-    height: auto;
-
-    .right {
-      padding: 0.5rem 0.25rem 0 0;
-    }
-
-    .meter {
-      background-color: #0b8c61;
-      padding: 0.5rem 0.25rem;
-      border-right: 1px solid #539f92;
-    }
+  ul, li {
+    list-style: none;
+    margin: 0;
+    font-size: 0.875rem;
   }
 
-  .expanded {
-    ul, li {
-      list-style: none;
-      margin: 0;
-      font-size: 0.875rem;
-    }
-
-    li {
-      background-color: #13bf85;
-      padding: 0 0.25rem;
-      margin-bottom: 2px;
-      color: white;
-    }
-
-    li.cost {
-      background-color: #0b8c61;
-    }
+  li {
+    border-bottom: .1em solid $grey-700;
+    padding: 0 0.25rem;
+    margin-bottom: 2px;
```

Comment:
IMO we should stick to either `px` or `rem` and I lean towards the latter so it scales with the font size.

Reply:
I agree. But I didn't change this line, except for indentation.